### PR TITLE
refactor: rename "clerk" to "admin"

### PR DESCRIPTION
**Pull request by eventualbuddha**
Closes https://github.com/votingworks/complete-system-ms/issues/12

### CODE REVIEW

File: apps/module-smartcards/writeClerkCard.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 f.close()
 
 election_json_bytes = json.dumps(election).encode('utf-8')
-short_value = json.dumps({'t':'clerk', 'h': hashlib.sha256(election_json_bytes).hexdigest()})
+short_value = json.dumps({'t':'admin', 'h': hashlib.sha256(election_json_bytes).hexdigest()})
```
Comment (benadida) [inline apps/module-smartcards/writeClerkCard.py:14]: change the filename too? --> `writeAdminCard.py`

**Review by benadida — APPROVED** [review]

**Review by carolinemodic — APPROVED** [review]

**Review by beausmith — APPROVED** [review]
We should update a couple items here too: https://github.com/votingworks/module-smartcards/search?q=clerk

**Comment by eventualbuddha** [comment]
> We should update a couple items here too: https://github.com/votingworks/module-smartcards/search?q=clerk

Those should all have been included in this PR: https://github.com/votingworks/vxsuite/pull/67/files#diff-9ae18a256c8dff918dde526e55fd7aeb5d56f2e5daa2479e1fc2c079ea75e799

File: apps/module-smartcards/writeClerkCard.py (hunk 1 above)
Comment (eventualbuddha) [reply apps/module-smartcards/writeClerkCard.py:14]: Done.

**Pull request merged**